packages/tailwindcss/theme.css

```diff
@@ -434,12 +434,12 @@
 
   --default-transition-duration: 150ms;
   --default-transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
-  --default-font-family: var(--font-sans);
-  --default-font-feature-settings: var(--font-sans--font-feature-settings);
-  --default-font-variation-settings: var(--font-sans--font-variation-settings);
-  --default-mono-font-family: var(--font-mono);
-  --default-mono-font-feature-settings: var(--font-mono--font-feature-settings);
-  --default-mono-font-variation-settings: var(--font-mono--font-variation-settings);
+  --default-font-family: --theme(--font-sans);
+  --default-font-feature-settings: --theme(--font-sans--font-feature-settings, normal);
+  --default-font-variation-settings: --theme(--font-sans--font-variation-settings, normal);
```

Comment:
Right now `--theme(…)`, just like `theme(…)`, errors when a value is not part of your theme and no fallback value is defined. 

We could potentially change this for non-inlined values but I think this is actually a good default, it's a feature that a CSS `var(…)` can not do but we can since we have access to the theme object. So, for now I decided to keep it even though it means we have to define a fallback for these variables (which makes sense since these are not defined within the default theme).

Hmm now that I think of this it probably means that `--default-font-family: --theme(--font-sans);` might also need a default value since otherwise if you unset the `--font-*` namespace, this will not compile. 🤔